frontend/institution/managementMembersController.js

```diff
@@ -271,6 +259,42 @@
             (field.email === empty_email.email) ? removeField(field) : addField();
         };
 
+        manageMemberCtrl.removePendingAndMembersEmails = function removePendingAndMembersEmails(emails) {
+            var requestedEmails = manageMemberCtrl.requests
+                .map(request => {
+                    return request.institutional_email;
+                });
+
+            var invitedEmails = manageMemberCtrl.sent_invitations
+                .map(invite => {
+                    return invite.invitee;
+                });
+
+            var membersEmails = [].concat.apply([], manageMemberCtrl.members
+                .map(member => {
+                    return member.email;
+                }));
+
+            var emailsNotMembersAndNotInvited = emails.filter(email =>
+                    !invitedEmails.includes(email) && !membersEmails.includes(email) 
+                        && !requestedEmails.includes(email));
+
+            return (emailsNotMembersAndNotInvited[0] === undefined) ? [] :  _.uniq(emailsNotMembersAndNotInvited);
```

Comment:
Why is necessary to verify if position 0 is undefined?

Reply:
Existe um problema, quando não encontra nenhum elemento retorna undefined e o length disso é 1.